newIDE/app/src/GameDashboard/index.js

```diff
@@ -0,0 +1,562 @@
+        return;
+      }
+
+      const fetchAuthenticatedData = async () => {
+        const [
+          feedbacks,
+          builds,
+          gameRollingMetrics,
+          leaderboards,
+          recommendedMarketingPlan,
+        ] = await Promise.all([
+          listComments(getAuthorizationHeader, profile.id, {
+            gameId: game.id,
+            type: 'FEEDBACK',
+          }),
+          getBuilds(getAuthorizationHeader, profile.id, game.id),
+          getGameMetricsFrom(
+            getAuthorizationHeader,
+            profile.id,
+            game.id,
+            oneWeekAgo.current.toISOString()
+          ),
+          listGameActiveLeaderboards(
+            getAuthorizationHeader,
+            profile.id,
+            game.id
+          ),
+          getRecommendedMarketingPlan(getAuthorizationHeader, {
+            gameId: game.id,
+            userId: profile.id,
+          }),
+          fetchGameFeaturings(),
+        ]);
+        setFeedbacks(feedbacks);
+        setBuilds(builds);
+        setGameMetrics(gameRollingMetrics);
+        setLeaderboards(leaderboards);
+        setRecommendedMarketingPlan(recommendedMarketingPlan);
+      };
+
+      fetchAuthenticatedData();
+    },
+    [getAuthorizationHeader, profile, fetchGameFeaturings, game.id]
+  );
+
+  const onClickBack = React.useCallback(
+    () => {
+      if (currentView === 'details') {
+        onBack();
+      } else {
+        setCurrentView('details');
+      }
+    },
+    [currentView, onBack, setCurrentView]
+  );
+
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <>
+          <Column noMargin expand>
+            <Line>
+              <TextButton
+                onClick={onClickBack}
+                icon={<ArrowLeft fontSize="small" />}
+                label={
+                  currentView === 'details' ? (
+                    <Trans>Back</Trans>
+                  ) : (
+                    <Trans>Back to {game.gameName}</Trans>
+                  )
+                }
+              />
+            </Line>
+            {currentView === 'feedback' ? (
+              <GameFeedback
+                authenticatedUser={authenticatedUser}
+                game={game}
+                i18n={i18n}
+              />
+            ) : currentView === 'builds' ? (
+              <Builds
+                game={game}
+                authenticatedUser={authenticatedUser}
+                onGameUpdated={_onGameUpdated}
+              />
+            ) : currentView === 'analytics' ? (
+              <GameAnalyticsPanel
+                game={game}
+                gameMetrics={gameRollingMetrics}
+                gameFeaturings={gameFeaturings}
+                fetchGameFeaturings={fetchGameFeaturings}
+                recommendedMarketingPlan={recommendedMarketingPlan}
+              />
+            ) : currentView === 'leaderboards' ? (
+              <LeaderboardAdmin gameId={game.id} onLoading={() => {}} />
```

Comment:
should the onLoading be plugged to something?

Reply:
I added a comment to say we don't need it